data/WillBeebe.json

```
@@ -36,7 +35,7 @@
       "icon": "FaLinkedin"
     },
     {
```

[Proofreading] This section is missing the url and name fields for LinkedIn. Check out the [quickstart on Linkfree's website to learn how to add this information](https://linkfree.io/docs/quickstart). 
```suggestion
      "icon": "FaLinkedin"
    },
    {
```

---

@CBID2 there have not been changes to this line and the LinkedIn link object has a URL...

```js
     {
      "name": "Let's connect on LinkedIn",
      "url": "https://www.linkedin.com/in/will-beebe/",
       "icon": "FaLinkedin"
     },
```

Therefore this is incorrect and I will mark as resolved.